.changeset/bitter-cameras-give.md

```diff
@@ -0,0 +1,10 @@
+---
+'unuse-angular': patch
+'unuse-react': patch
+'unuse-solid': patch
+'unuse-vue': patch
+'unuse': patch
+---
+
+fix: isUnRef recognize also unSignal, unComputed and getter functions
+fix!: remove internal override functions from public API
```

Comment:
Usually we add two changesets for multi changes.

By the way, this can be done at https://github.com/un-ts/unuse/pull/31#issuecomment-2997054711

Reply:
I was not aware of that, fixed it and wont happen in future 👍